src/main/resources/db/migration/0300/V302__DeliverableInternalInterestMapping.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE accelerator.deliverable_categories
+    ADD COLUMN internal_interest_id INTEGER REFERENCES accelerator.internal_interests ON DELETE CASCADE;
```

Comment:
ON DELETE CASCADE seems dangerous here -- I think we want to block deleting an interest that maps to a deliverable category, rather than deleting the category (and anything that refers to the category with an ON DELETE CASCADE) when the interest is deleted.